config/testgrids/conformance/conformance-all.yaml

```diff
@@ -155,13 +154,6 @@ dashboards:
 - name: conformance-vsphere
 - name: conformance-hack-local-up-cluster
 
-# ppc64le dashboard
```

Comment:
You shouldn't remove the conformance entries.

Reply:
@mkumatag Do you mean to have the dashboard tab for conformance run at both places? In `provider-ibm` group and in `conformance-all` group.
We may have the test_group details (gcs details) defined only once in config.yaml under ibm folder.

Reply:
yes, the conformance tests should be there in the `conformance-all` group for sure.